tests/Doctrine/Tests/ODM/PHPCR/DocumentManagerTest.php

```diff
@@ -54,7 +54,7 @@ public function testFindTranslation()
      */
     public function testNewInstanceFromConfiguration()
     {
-        $session = $this->getMock('PHPCR\SessionInterface');
+        $session = $this->getMockBuilder('PHPCR\SessionInterface')->getMock();
```

Comment:
the recommended way to do this is to use the new `createMock` method, afaik. the difference is that createMock disables the original constructor. only when we need additional things from the mock builder should we use getMockBuilder.